From e6e1cfc49a8502b476038245b8de44e4f77e9169 Mon Sep 17 00:00:00 2001
From: TungDang <tung.dang.jy@rvc.renesas.com>
Date: Sun, 21 Aug 2016 10:09:32 +0700
Subject: [PATCH 9/9] waylandsink: Preliminary for the gstdmabuf allocator registration to the buffer pool

This patch update gstreamer-allocators in PKG_CHECK_MODULE to ensure correct setting as commit
332eef9492f80d84351320ee41c6b686528f5f80 version 1.2.3

Porting from a patch in Linux 3.10 environment made by Kazunori Kobayashi <kkobayas@igel.co.jp>

Signed-off-by: TungDang <tung.dang.jy@rvc.renesas.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index e9a38c1..6fd13f0 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1977,7 +1977,7 @@ AG_GST_CHECK_FEATURE(WAYLAND, [wayland sink], wayland , [
   PKG_CHECK_MODULES(WAYLAND, wayland-client >= 1.4.0 wayland-scanner, [
     HAVE_WAYLAND="yes" ], [ HAVE_WAYLAND="no"
   ])
-  PKG_CHECK_MODULES(WAYLAND_KMS, libdrm libkms wayland-kms,
+  PKG_CHECK_MODULES(WAYLAND_KMS, libdrm libkms wayland-kms gstreamer-allocators-1.0,
     [HAVE_WAYLAND_KMS="yes"
         AC_DEFINE(HAVE_WAYLAND_KMS, 1, [Define if wayland KMS protocol is available])],
        [HAVE_WAYLAND_KMS="no"])
--
1.9.1
